City-of-Helsinki/drupal-module-helfi-tpr
# UHF-7334: Support for OPENING_HOUR_OBJECT connections

## How to install

- `composer require drupal/helfi_tpr:dev-UHF-7334`
- `drush cr`

## How to test

- Open `/tpr-unit/57184` page and make sure only "Voimassa toistaiseksi" opening hour is visible: 
![image](https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/assets/771113/7ca3e909-f7b1-419f-bfe3-bf36c10c2e0c)

- Run `drush mim tpr_unit 57184:sv,57184:en,57184:fi --update`
- Refresh the page and make sure "Testiaukiolo" is visible: 
![image](https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/assets/771113/1516fed8-bc5b-4cc9-8a15-2ef606ed34f2)
